src/pages/Dashboard/TeamOverview.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { Box, Button, Center, Flex, Heading, Spacer } from '@chakra-ui/react';
+
+const GET_TEAM_INFO = gql`
+  query getTeam($id: Int!) {
+    getTeam(id: $id) {
+      name
+      organization
+      id
+      amountRaised
+      itemsSold
+    }
+  }
+`;
+
+const StorefrontButton = () => {
+  return <Button w="199px">Share Storefront</Button>;
+};
+
+const SalesInfo = props => {
+  return (
+    <Box mr="100px">
+      <Heading textTransform="uppercase" size="subtitle" color="gray.500" mb="8px">
+        {props.description}
+      </Heading>
+      <Heading size="h2">{props.amount}</Heading>
+    </Box>
+  );
+};
+
+const NoSales = () => {
+  return (
+    <Box w="100%" h="300px" bg="background.primary">
+      <Center w="100%">
+        <Heading size="h4" mt="69px" mb="24px">
```

Comment:
nice! should use the `as="h4"` (or equivalent for other headings) for semantic purposes

Reply:
hmm i thought `size="h4"` was the correct way for headings? it doesn't work if i use `as="h4"`